src/notifications.c

```diff
@@ -17,9 +17,10 @@
 #include "dictionary.h"
 #include "slot_ranges.h"
 #include "asm_state_machine.h"
+//TODO ASM: Review this, maybe is worth duplicating the logic or moving to utils?

```

Comment:
leftovers?

Reply:
Thinking it better, if we ever have to remove coordinator into a separate process, the Topology Update would be handled differently and this would be very naturally changed, so no point on changing now